sklintyg/frontend
# Run prettier independent from linter

It's not recommended to use [eslint-plugin-prettier](https://prettier.io/docs/en/integrating-with-linters.html). Instead, it's recommended to use prettier --check to verify formatting is correct. This will simplify prettier integration a bit and hopefully fix the issues we are facing with ESLint since changing to pnpm.

* Update prettier to latest version
* Update ESLint to latest version
* Add step to GitHub actions for verifying formatting
* Add prettier to pre-commit hook
* Fix pre-commit issue on Windows
* Patch current version of ids-core (issue with loading unlisted dependency lit-html)
* Remove eslint-plugin-prettier
